cmd/internal/cli/singularity.go

```diff
@@ -208,6 +233,8 @@ var SingularityCmd = &cobra.Command{
 func persistentPreRunE(cmd *cobra.Command, _ []string) error {
 	setSylogMessageLevel()
 	setSylogColor()
+	confDir := getConfDir(CurrentUser.Username)
```

Comment:
Nitpick: I think you should use `syfs.ConfigDir()` here since these functions appear to do the same thing

Reply:
Not a nitpick at all.  Nice catch!